lnwallet/commitment.go

```diff
@@ -652,10 +660,10 @@ func (c *CommitAuxLeaves) ForRemoteCommit() CommitAuxLeaves {
 // TODO(roasbeef): move into pkg for custom chans?
 type AuxLeafStore interface {
 	// FetchLeavesFromView attempts to fetch the auxiliary leaves that
-	// correspond to the passed aux blob, and pending fully evaluated HTLC
-	// view.
+	// correspond to the passed aux blob, and pending original (unfiltered)
+	// HTLC view.
 	FetchLeavesFromView(chanState *channeldb.OpenChannel, prevBlob tlv.Blob,
-		view *HtlcView,
+		unfilteredView *HtlcView,
```

Comment:
I think it still wants a partially filtered view, as we want to make sure that's what we're covering in the sig is also reflected here. Still checking out the later commits.